### DISCUSSION TITLE
CryptoTokenKitError -3 when trying to approve login requests

### DISCUSSION THREAD

**Discussion by Kevin-Andries** [community/community#103764] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.147.0 (121573070)
Device: iPhone 11 (iOS 17.2.1)
</details>

Hi,

I have been having a terrible issue. I can't login anymore into my GitHub account because of an error happening when I try to approve the login request from the mobile app (see screenshot below).
I contacted the support but I didn't get any answer. So I'm locked outside of my account (except on mobile). I can't work or download my project code. I tried reinstalling the app but still no success.

I would appreciate any help to fix this please.

![image](https://github.com/community/community/assets/70948628/1da83ac0-4347-4cc9-9cf3-e560610b7e21)

Thanks

**Comment by elisealix22** [comment]
@Kevin-Andries thanks for reaching out! When does that error show for you? Are you trying to authenticate into a second account? Is it after a 2-factor log in attempt for example? Any more details that you can provide would be helpful!

**Reply by Kevin-Andries** [reply]
@elisealix22 Hi! It happens whenever I try to log in on my laptop. I type my username and password, then a prompt appears asking me to approve the sign in request on the mobile app, with eventually a 2-digit code to type as well. I then open the mobile app, and a pop up appears to approve or reject the sign in request. When I do so, (approve and/or type the code) the pop up disappears in the app and the message on the screenshot appears. Nothing happens on my laptop. I don't have a second account or anything, I always used only this one.
I tried to explain as precisely as I could, let me know if you need anything else.
Thank you for your help, because I'm totally stuck!

**Reply by elisealix22** [reply]
Thank you so much for this information! It is extremely helpful for us to figure out what is going on. I opened an internal issue and we are looking into it. I'm really sorry you're running into this issue.

**Reply by Kevin-Andries** [reply]
@elisealix22 Hi, do you have any news about this issue please? I still can't access my GitHub account because of 2FA being enabled on this app :/ Thanks a lot

**Reply by Kevin-Andries** [reply]
Hi @elisealix22,
Just to let you know that I managed to log in using Safari instead of Chrome. With Safari, the error in the mobile app didn't show up. Quite strange...
Hope this helps someone else.

**Reply by elisealix22** [reply]
@Kevin-Andries Thank you for letting us know! I updated the team that it could be browser-specific. We are still looking into it and this is really helpful information.

**Comment by ruhail12** [comment]
Efficiently written information. It will be profitable to anybody who utilizes it, counting me. Keep up the good work. For certain I will review out more posts day in and day out.	<a href="https://www.1skupaut.pl/swietochlowice/">skup aut swietochlowice</a>
